RELEASE.md

```diff
@@ -1,7 +1,28 @@
+# Release 6.0
+
+Apollo 6.0 incorporates new deep learning models to enhance the capabilities for certain Apollo modules. This version works seamlessly with new addtions of data pipeline services to better serve Apollo developers. Apollo 6.0 is also the first version to integrate certain features as a demonstration of our continuous exploration and experimentation efforts towards driverless technology.
+
+## Major Features and Improvements
+* Upgraded Deep Learning Models
+    * PointPillars based obstacle detection model
+    * Semantic map based pedestrian prediction model
+    * Learning based trajectory planning model
+* Brand New Data Pipeline Services
+    * Low speed obstacle prediction model training service with semantic map support
+    * PointPillars based obstacle detection model training service
+    * Control profiling service
+    * Vehicle dynamic model training service
+    * Open space planner profiling service
+    * Complete control parameter auto-tune service
+* Driverless Research
+    * Remote control interface with DreamView integration
+    * Audio bssed emergency vehicle detection system
+
+**[Note]** All models and methodogies included in Apollo 6.0 are for research purposes only. Productized and commercial uses of these models are **NOT** encouraged, and it is at your own risk. Please be cautious to try Apollo 6.0 with enough safety protection mechanmism and provide your feedback so that we can continuously improve the models.
+
```

Comment:
`and provide your feedback ...` -> and any feedback is welcome for us to continuously improve the models